mmv1/third_party/terraform/website/docs/d/apphub_discovered_workload.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+subcategory: "Apphub"
+description: |-
+  Get information about a discovered workload.
+---
+
+# google\_apphub\_discovered_workload
+
+Get information about a discovered workload from its uri.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_discovered_workload" "my-workload" {
+  location = "my-location"
+  workload_uri = "my-workload-uri"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `workload_uri` - The uri of the workload.
+* `location` - The location you want to get the discovered worklaod from
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `discovered_workload` - Represents a network/api interface that exposes some functionality to clients for consumption over the network. A `discovered_workload` object would contain the following fields:-
+
+  - `name` - Resource name of a Workload. Format: "projects/{host-project-id}/locations/{location}/applications/{application-id}/workloads/{workload-id}".
+
+  - `workload_reference` - Reference to an underlying networking resource that can comprise a Workload. A `workload_reference` object would contain the following fields:-
+    - uri - The underlying resource URI.
+    - path - Additional path under the resource URI.
+
+  - `workload_properties` - Properties of an underlying compute resource that can comprise a Workload. A `workload_properties` object would contain the following fields:-
+      - gcp_project - The service project identifier that the underlying cloud resource resides in.
+      - location - The location that the underlying resource resides in.
+      - zone - The location that the underlying resource resides in if it is zonal.
```

Comment:
Can you please follow this example for the nested object doc? Thanks.

https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/d/cloud_identity_group_membership.html.markdown?plain=1#L29

Reply:
Done.